core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -27,50 +27,60 @@
 {% endblock header_js %}
 
 {% block content %}
-  <div ng-controller="CollectionEditor">
-    <div ng-if="collection">
-      <!-- Collection details -->
-      <collection-details-editor-directive collection="collection">
-      </collection-details-editor-directive>
-      <a ng-href="/collection/<[collection.getId()]>" target="_blank">Go to Collection</a>
+  <div class="oppia-page-cards-container">
```

Comment:
Why is this being done? There is separate work for cleaning up the collection editor and finalizing its UI. This seems like something that would be better to do in #1748 (I'm trying to reduce the merge conflicts on this file, which seem high at the moment).